src/pages/carbon-offsetting.jsx

```diff
@@ -0,0 +1,564 @@
+      }}
+    />
+    <Container>
+      <Row>
+        <Col>
+          <h1>Is "Carbon Offsetting" Good or Bad?</h1>
+
+          <p>
+            Depending on who you talk to, it might be touted as the solution to
+            all the worlds problems, greenwashing, fraud, a distraction which
+            perpetuates the status quo and hampers real climate action, or a
+            complex grey area which needs a bit of explanation to understand
+            properly. Let's explain why people are so divided.
+          </p>
+
+          <p>
+            Carbon offsetting is generally the idea of a person, business, or
+            government, paying some amount of money to an organization who is
+            working on a project to reduce emissions, or capture existing
+            emissions. These activities could be all sorts of things:
+          </p>
+
+          <ul>
+            <li>Planting trees to suck up CO2 over time</li>
+            <li>Protecting woodlands from being cut down</li>
+            <li>
+              Handing out efficient cooking stoves in developing countries to
+              avoid trees being cut down and burned
+            </li>
+            <li>
+              Funding renewable energy projects to reduce/remove reliance on
+              fossil fuels
+            </li>
+            <li>Capturing methane. gas emissions from sites like landfills</li>
+          </ul>
+
+          <p>
+            In theory, carbon offsetting is a great idea, but any good can be
+            corrupted by greed.
+          </p>
+
+          <h2 id="airlinesprofitingoffthestatusquo">
+            Airlines profiting off the status quo
+          </h2>
+
+          <p>
+            Various airlines picked up the idea and offer their own carbon
+            offsetting programs. They charge a small fee on top of the ticket
+            price, but usually provide very little insight into where that money
+            is going, what it's doing, how much goes in admin costs.
+          </p>
+
+          <p>For example, Ryan Air were caught in a lie.</p>
+
+          <blockquote>
+            <p>
+              While [Ryan Air] do contribute to carbon offset programs, The BBC
+              determined that in order to completely offset the fossil fuels
+              their flights were burning, the airline would have to plant and
+              cover 12 percent of the United Kingdom in new trees. Their program
+              only planted seven acres of new forest. -- Source:{' '}
+              <a href="https://www.travelandleisure.com/airlines-airports/easyjet-offset-carbon-emissions-airplanes-environmental-impact">
+                travelandleisure.com
+              </a>
+            </p>
+          </blockquote>
+
+          <p>
+            Planting only seven acres of "new forest" - which is basically 7
+            soccer pitches - will do basically nothing for the environment,
```

Comment:
I feel like saying 'planting 7 acres of trees will do basically nothing' suggests that small scale reforestation isn't a worthwhile pursuit.  Improve this.